sufia-models/app/models/datastreams/fits_datastream.rb

```diff
@@ -144,42 +144,7 @@ def self.xml_template
     http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd",
                version: "0.6.0",
                timestamp: "1/25/12 11:04 AM") {
-        xml.identification {
```

Comment:
Whoops! I misread the commit. You are right. This is the template.  Yeah, I don't see any reason for us to generate a fully formed template. :+1: